package/piksi_ins/overlay/usr/bin/start_pose_daemon

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+key_index=1
+
+b=/usr/bin
+l=/usr/lib
+
+caps="cap_ipc_lock,cap_sys_rawio+ep"
+
```

Comment:
I have no idea from inspection what is happening here.  Maybe a comment or two could help the next engineer?

Reply:
I'll add a comment but basically they are limited root privileges (capabilities) `cap_ipc_lock` allows is to run mmap on a region outside of it's virtual memory and `cap_sys_rawio` allows it to open `/dev/mem`.  The `+ep` string turns the permissions on for the process.

Reply:
Fixed